csrc/scheduler/tools/loop_domain_scheduler.h

```diff
@@ -22,7 +22,8 @@ namespace scheduler_tools {
 // each tensor by replaying exprs found in the Exact ValGraph.
 void scheduleLoopDomainsLike(
     const std::vector<TensorView*>& tvs,
-    const std::vector<IterDomain*>& ref_loop_dom);
+    const std::vector<IterDomain*>& ref_loop_dom,
+    bool update_mode = false);
```

Comment:
We should document `update_mode` in the comment here.

Reply:
Would a better name be `update_loop_domain_only`? That name implies that we will not be modifying the logical domain to place the loop as a producer of logical.